format/SparseTensor.fbs

```diff
@@ -0,0 +1,52 @@
+include "Schema.fbs"
+
+namespace org.apache.arrow.flatbuf;
+
+/// Coodinate format.
+table SparseTensorIndexCOO {
+  /// COO's index list are represented as a NxM matrix,
+  /// where N is the number of non-zero values,
+  /// and M is the number of dimensions of a sparse tensor.
+  /// indicesBuffer stores the location and size of this index matrix.
+  /// The type of index value is long, so the stride for the index matrix is unnecessary.
+  indicesBuffer: Buffer
+};
+
+/// Compressed Sparse Row format, that is matrix-specific.
+table SparseMatrixIndexCSR {
+  /// This array represents the range of the rows.
+  /// The ith row spans from indptr[i] to indptr[i+1] in the data.
+  /// The length of this array is 1 + (the number of rows).
+  indptr: [long]
```

Comment:
The name `indptr` comes from scipy's CSR implementation.
I want to replace this with a more suitable name.